docs/source/api.rst

```diff
@@ -219,3 +226,14 @@ Preprocessing
     :toctree: api/
 
     get_path
+
+Smoothing
+~~~~~~~~~
+.. currentmodule:: climpred.smoothing
+
+.. autosummary::
+    :toctree: api/
+
+    temporal_smoothing
+    spatial_smoothing_xesmf
+    _reset_temporal_axis
```

Comment:
Should this be in the API? The underscore hints that the user shouldn't have to touch it.

Reply:
deleted